docs/error.md

```diff
@@ -145,21 +145,8 @@ That would generate the following code after validation:
 
 Method | Description | Default
 -------|-------------|---------
+`config(FormModelInterface $formModel, string $attribute, array $attributes = [])` | Configures the widget. |
```

Comment:
See https://github.com/yiisoft/form/pull/144/files#r745983076

Reply:
But that conflicts with https://github.com/yiisoft/form/pull/146/files#diff-b2dbc04239858c959223583f766058bc1685ade0b8ff4c9278cd3d30866bd5aaR119 :(